app-shells/mcfly/mcfly-0.9.3.recipe

```diff
@@ -56,15 +56,15 @@ BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	"
 BUILD_PREREQUIRES="
-	cmd:cargo
+	cmd:cargo$secondaryArchSuffix
 	cmd:gcc$secondaryArchSuffix
-	cmd:rustc
+	cmd:rustc$secondaryArchSuffix
 	"
 
 BUILD()
 {
-	cargo fetch --locked --target "$(rustc -vV | sed -n 's/host: //p')"
-	cargo build --release
+	cargo$secondaryArchSuffix fetch --locked --target "$(rustc$secondaryArchSuffix -vV | sed -n 's/host: //p')"
+	cargo$secondaryArchSuffix build --release
```

Comment:
These are not needed, I guess haikuporter switches to "arch x86" during the build. (at least worked without those here).